plugins/handler/plugin_api_handler.go

```diff
@@ -14,6 +14,15 @@
 
 package handler
 
+const (
+	NetworkServiceResource         = "networkservice"
+	NetworkServiceChannelResource  = "networkservicechannel"
+	NetworkServiceEndpointResource = "networkserviceendpoint"
+	HandlerCreate                  = "create"
+	HandlerDelete                  = "delete"
+	HandlerUpdate                  = "update"
+)
```

Comment:
These are probably not needed any more, right?

Reply:
D'oh! You're right, I'll remove them, good catch!